File: back/src/main/java/perso/arcade/model/dto/ModifSpriteDto.java
package perso.arcade.model.dto;

public class ModifSpriteDto {

    private Long id;
    private String newName;
    private float scale;

    public float getScale() {
        return scale;
    }

    public void setScale(float scale) {
        this.scale = scale;
    }

    public String getNewName() {
        return newName;
    }

    public void setNewName(String newName) {
        this.newName = newName;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }
}

File: front/src/models/dtos/modifSpriteDto.ts
export default class ModifSpriteDto {
    oldName: string;
    newName: string;
    scale: number;

    constructor(oldName: string, newName: string, scale: number) {
        this.oldName = oldName;
        this.newName = newName;
        this.scale = scale;
    }
}
